src/oiiotool/oiiotool.cpp

```diff
@@ -1503,8 +1511,17 @@ class OpAttribSetter final : public OiiotoolOp {
         : OiiotoolOp(ot, opname, argv, 1)
     {
         inplace(true);  // This action operates in-place
-        attribname = args(1);
-        value      = (nargs() > 2 ? args(2) : "");
+        auto options  = ot.extract_options(opname);
+        bool fromfile = options.get_int("fromfile", 0);
+        if (!fromfile) {
+            attribname = args(1);
+        } else {
+            // handle erase attribute using regex text file case
+            // defined it outside to avoid magic number, still not very elegant solution wise
```

Comment:
I don't understand this comment. Magic number?

Reply:
Feel free to disregard! Second sentence was leftover commentary from the old old prototype design, doesn't apply here at all after design 2. Will be removing that